packages/blade/src/components/Indicator/_decisions/decisions.md

```diff
@@ -0,0 +1,111 @@
+# Indicator
+
+Indicators describe the condition of an entity. They can be used to convey semantic meaning, such as statuses and semantical-categories.
+
+This document outlines the API of `Indicator` component.
+
+<img src="./indicator-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Indicator](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8224%3A1)
+
+## API
+
+Sample usecase:
+
+<img src="./with-label.png" width="380" alt="" />
+
+Sample usage:
+
+```jsx
+<Indicator
+  intent="positive"
+  label="Active | xyz@email.com"
+  accessibilityLabel="Active for xyz@email.com"
+/>
+```
+
+### Indicator
+
+We'll expose an `Indicator` component with the following API:
+
+| Prop               | Type                                                      | Default     | Description                                    | Required |
+| ------------------ | --------------------------------------------------------- | ----------- | ---------------------------------------------- | -------- |
+| accessibilityLabel | `string`                                                  | `undefined` | Sets the aria-label for screen readers         | ♿️      |
+| label              | `string`                                                  | `undefined` | Shows a text label beside the circle indicator |          |
```

Comment:
Why use `label` as a prop and not `children` for rendering the label instead? That's what we're doing on Badge, Radio, Checkbox, etc. Label associated with the component is taken as `children`.

Reply:
Was thinking of using children for the composition API (in alternatives section) earlier. But we can use `children` too to match other components

Reply:
Updated in https://github.com/razorpay/blade/pull/788/commits/dcd11dae93bf81e674aa49356a68f824cb34b2c1